src/epochs.lua

```diff
@@ -370,71 +359,62 @@ end
 --- @param timestamp number The timestamp in milliseconds
 --- @param blockHeight number The block height
 --- @param hashchain string The hashchain
---- @return Epoch|nil # The created epoch, or nil if an epoch already exists for the index
-function epochs.createEpoch(timestamp, blockHeight, hashchain)
+--- @return PrescribedEpoch | nil # The created epoch, or nil if an epoch already exists for the index
```

Comment:
Could also qualify that it'd be nil before epoch0 timestamp

Reply:
added checks for < 1 in https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f